recipes/hapog/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: hapog
+  version: 1.3.1
+
+source:
+  url: https://github.com/institut-de-genomique/HAPO-G/archive/refs/tags/1.3.1.tar.gz
```

Comment:
You could use templates here. Define the version on top and reuse the version number here.